### ISSUE TITLE
Photoshop plugin exports white page

### ISSUE THREAD

**Issue by piotrski** [avocode/avocode#1064]
![zrzut ekranu 2016-08-24 o 13 25 53](https://cloud.githubusercontent.com/assets/244174/17929121/bc5c52c6-69fe-11e6-9e95-595c5227187f.png)
It's strange because it looks ok in the browser. I'm using Photoshop plugin v.2.2.3

**Comment by connor-bean** [comment]
Hey @piotrski -

Thanks for the report. What version of Avocode are you running and what OS?

Could you send us the problematic design to team@avocode.com? Please mention this ticket number 1064 in your email to team@avocode.com

Thanks,
Connor, Avocode Support

**Comment by connor-bean** [comment]
Thanks @piotrski.

I received your email. We will follow up here once we have more information.

**Comment by bolekkerous** [comment]
It looks like there were artboards and layer comps used in the design. That's the reason the design is not showing correctly in Avocode. Unfortunately, Avocode doesn't support artboards nor layer comps yet.

There are already issues on our UserVoice forum, which can be upvoted, so we know what to prioritize in the future:

https://avocode.uservoice.com/forums/277080-feature-voting/suggestions/6992632-show-layer-comps

https://avocode.uservoice.com/forums/277080-feature-voting/suggestions/9428469-support-photoshop-artboards

We'd advise to ungroup artboards in the design and remove layer comps in the meantime. Once that is done, the design be displayed just fine.

**Comment by piotrski** [comment]
Thanks for the quick response, it helped. Maybe you should add warning when uploading design with artboarda and layer comps?
Is there some easy way to debug why design renders incorrectly? I have second one which renders as transparent canvas. It doesn’t have artbords nor layer comps. I just don’t want to bother you with such simple issues.

**Comment by bolekkerous** [comment]
There was a bug previously which may have caused the transparent canvas, but it should be fixed for a long time now. Please download the latest version of Avocode app:

https://manager.avocode.com/downloads/

I'll close this one as a duplicate - feel free to comment or create a new issue if you have any other problems. You can also hit us up at team@avocode.com with any issues :-)

As for the artboards issue, you can follow it on the original issue #374
